samples/snippets/create_multiple_timeseries_forecasting_model.py

```diff
@@ -0,0 +1,106 @@
+def test_multiple_timeseries_forecasting_model(random_model_id):
+    # [START bigquery_dataframes_bqml_create_data__set]
```

Comment:
As discussed in our 1:1, these region tags need to be globally unique, so let's borrow from the URL of the tutorial (https://cloud.google.com/bigquery/docs/arima-multiple-time-series-forecasting-tutorial) to construct these.

For example:

```suggestion
    # [START bigquery_dataframes_bqml_arima_multiple_step_2_visualize]
```

Reply:
yes, region tags will follow URL.

Reply:
Corrected